package.json

```diff
@@ -10,11 +10,10 @@
     "@ledgerhq/hw-transport-webusb": "^6.27.13",
     "@phosphor-icons/react": "^2.0.10",
     "@react-spring/web": "^9.6.1",
-    "@secretkeylabs/xverse-core": "6.0.1",
-    "@stacks/connect": "^6.10.2",
-    "@stacks/encryption": "4.3.5",
```

Comment:
It looks like we currently still include `@stacks/encryption` in `webpack/webpack.config.js`, so I think it would make sense to remove it from there as well